### DISCUSSION TITLE
PayPal Module: Authorize a payment and capture funds later

### DISCUSSION THREAD

**Discussion by djixas** [OpenMage/magento-lts#3940] [Extensions & Modules]
Hello,

Does OpenMage 19 support PayPal's Authorize a payment and capture funds later feature? I can't seem to find documentation

https://developer.paypal.com/docs/multiparty/checkout/standard/customize/auth-capture/

**Comment by jparker1986** [comment]
Yes this is a standard feature but you must manually "accept" the payment.
